### PR TITLE
doc: add a troubleshooting article about the missing configuration files

### PR DESCRIPTION
**Pull request by annastuchlik**
Fix https://github.com/scylladb/scylladb/issues/11598

This PR adds the troubleshooting article submitted by @syuu1228 in the deprecated _scylla-docs_ repo, with https://github.com/scylladb/scylla-docs/pull/4152.
I copied and reorganized the content and rewritten it a little according to the RST guidelines so that the page renders correctly.

@syuu1228 Could you review this PR to make sure that my changes didn't distort the original meaning?

### CODE REVIEW

File: docs/troubleshooting/missing-dotmount-files.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+the ``/etc/systemd/system/var-lib-scylla.mount`` and ``/etc/systemd/system/var-lib-systemd-coredump.mount`` are 
+deleted by RPM.
+
+To avoid losing the files, the :doc:`upgrade procedure </upgrade/upgrade-to-enterprise/upgrade-guide-from-5.0-to-2022.1/upgrade-guide-from-5.0-to-2022.1-rpm/>` 
+includes a step to backup the .mount files:
+
+
+.. code-block:: console
+
+    for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ) /etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; do sudo cp -v $conf $conf.backup-4.3; done
```
Comment (tzach) [inline docs/troubleshooting/missing-dotmount-files.rst:20]: @syuu1228 why 4.3 here in `$conf.backup-4.3;` ?

File: docs/troubleshooting/missing-dotmount-files.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ 
+However, if you don't backup the .mount before the upgrade, the files may be lost.
+
+
+Solution
+^^^^^^^^
+
+To fix this problem, you need to manually restore .mount files.
```
Comment (tzach) [inline docs/troubleshooting/missing-dotmount-files.rst:28]: To validate, the solution is for the case one did **not** followed the procedure above of backup the .mount ?

**Comment by denesb** [comment]
Please change the PR title to something more descriptive.

File: docs/troubleshooting/missing-dotmount-files.rst (hunk 2 above)
Comment (syuu1228) [reply docs/troubleshooting/missing-dotmount-files.rst:28]: @tzach Yes, it's just for the case who didn't backup .mount.

File: docs/troubleshooting/missing-dotmount-files.rst (hunk 1 above)
Comment (tzach) [reply docs/troubleshooting/missing-dotmount-files.rst:20]: +1

or clarify this is an example, and one should use the relevant release number

Comment (annastuchlik) [reply docs/troubleshooting/missing-dotmount-files.rst]: @tzach @syuu1228 I've modified that section to improve clarity. Could you re-review it?

**Review by tzach — APPROVED** [review]
LGTM from doc prespective, but I didnt test the procedure.
@yaronkaikov can you please take a look ?

**Comment by tzach** [comment]
@syuu1228 ping

**Comment by annastuchlik** [comment]
@syuu1228 @yaronkaikov Could you have a final look at the changes so that this PR can be merged?

**Comment by yaronkaikov** [comment]
@syuu1228 Can you help with this?

**Comment by annastuchlik** [comment]
@syuu1228 Could you do the final review? Most of the content is what you proposed yourself, so it shouldn't take long.

**Review by syuu1228 — APPROVED** [review]
LGTM

**Pull request closed without merging**